### ISSUE TITLE
gpflow 2.0 release

### ISSUE THREAD

**Issue by a-z-e-r-i-l-a** [GPflow/GPflow#1018]
When does will the compatible version of gpflow with tensorflow 2.0 be released. Is it safe to use the current gpflow 2.0 branch?

**Comment by awav** [comment]
As soon as TF2.0 official release will be announced we will make a GPflow 2.0 release. You can use `awav/gpflow-2.0`, in which feature of GPflow 2.0 are you interested in?

**Comment by a-z-e-r-i-l-a** [comment]
I am interested in learning an embedding of an input space with a shallow neural network based on minimizing the negative marginal log likelihood similar to this [paper],(https://arxiv.org/abs/1402.5876), 

So far I have [a few lines of code that](https://github.com/a-z-e-r-i-l-a/DeepGP/blob/master/19.08.13%20TF%202.0%20.ipynb) takes the gradient of the negative-log-likelihood w.r.t. the learnable weights (in this case the length scales) and take gradient decent steps for minimizing it. Although, I expect to be, in the worst case, overfitting the data, my loss on the training set becomes actually more which I am not sure what may be the reason.

**Comment by ChunchaoPeter** [comment]
I am interested in the Coregionalisation model in Multi-output GPs. Do you plan to build the coregionalisation kernel?

Thanks

**Comment by markvdw** [comment]
The coregionalisation kernel is currently in GPflow. A version that works with inducing points is called `LinearCoregionalization`. Check out the multi-output notebook to see how to do sparse inference!

We welcome contributions for some extensions.

**Comment by st--** [comment]
The `develop` branch now contains a tensorflow 2.0-compatible version, and there is a 2.0-rc1. We aim to be making a full release shortly. If you've got anything you think needs doing before then please add it to the corresponding project board.
